docs/development/contributing.rst

```diff
@@ -0,0 +1,48 @@
+==========================================
+Contributing to the Plone Intranet Project
+==========================================
+
+.. admonition:: Description
+
+   How to develop and submit code for the Plone Intranet Project.
+
+.. contents:: :local:
+
+
+Reaching the Plone Intranet team
+================================
+
+To reach the Plone Intranet development team for any questions please contact
+
+* `Plone Intranet Developers mailing list <mailto:ploneintranet-dev@groups.io>`_
+
+* *#plone.intranet* IRC channel on irc.freenode.net
+
+License
+=======
+
+.. note::
+
+   License goes here
+
+Definition of Done
+==================
+
+In order for a pull request to be accepted
+the following development and style guidelines must be adhered to:
+
+ * The `continuous integration <http://jenkins.ploneintranet.org>`_ build must be
+ passing
+ * Any contributed code, must have 100% test coverage
```

Comment:
@mattss I think "Any contributed code must have 100% test coverage" says exactly that.